can't start docs

I am generating a victory component with `yo` but I cannot start the docs:

``` bash
builder run docs-dev

Error: Unable to find task for: docs-dev
    at Task.getCommand (/Users/ph185068/projects/di/leapfrog/poc/components/button/node_modules/builder/lib/task.js:88:11)
    at Task.run (/Users/ph185068/projects/di/leapfrog/poc/components/button/node_modules/builder/lib/task.js:124:19)
    at Task.execute (/Users/ph185068/projects/di/leapfrog/poc/components/button/node_modules/builder/lib/task.js:187:21)
    at Object.<anonymous> (/Users/ph185068/projects/di/leapfrog/poc/components/button/node_modules/builder/bin/builder.js:22:6)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:138:18)
```

I tried updating to:

``` bash
"builder-react-component": "0.1.2",
"builder-react-component-dev": "0.1.2",
```

Still no joy. Tried running command manually:

``` bash
webpack-dev-server --port 3000 --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.dev.js --content-base docs
module.js:340
    throw err;
    ^

Error: Cannot find module 'esprima'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/ph185068/projects/di/leapfrog/poc/components/button/node_modules/webpack/lib/Parser.js:5:15)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

Installed `esprima`, then `builder-victory-component` try again:

``` bash
$ webpack-dev-server --port 3000 --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.dev.js --content-base docs


ERROR in ./docs/docs.jsx
Module not found: Error: Cannot resolve module 'ecology' in /Users/ph185068/projects/di/leapfrog/poc/components/button/docs
 @ ./docs/docs.jsx 27:15-33

ERROR in ./docs/docs.jsx
Module not found: Error: Cannot resolve module 'react-docgen' in /Users/ph185068/projects/di/leapfrog/poc/components/button/docs
 @ ./docs/docs.jsx 35:19-42

ERROR in ./docs/docs.jsx
Module not found: Error: Cannot resolve module 'formidable-landers' in /Users/ph185068/projects/di/leapfrog/poc/components/button/docs
 @ ./docs/docs.jsx 39:25-54

ERROR in ./docs/docs.jsx
Module not found: Error: Cannot resolve module 'raw' in /Users/ph185068/projects/di/leapfrog/poc/components/button/docs
 @ ./docs/docs.jsx 57:20-49

ERROR in ./docs/docs.jsx
Module not found: Error: Cannot resolve module 'raw' in /Users/ph185068/projects/di/leapfrog/poc/components/button/docs
 @ ./docs/docs.jsx 58:31-72
```

``` bash
$ npm install ecology react-docgen formidable-landers raw-loader
$ webpack-dev-server --port 3000 --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.dev.js --content-base do
```

Runs but get the following error in browser window:

`Oops! Something is broken or JavaScript is not enabled.`

and console:

`Uncaught Error: No suitable component definition found.`

Tried to publish component with:

``` bash
npm publish
```

But get following error when I try to install elsewhere:

``` bash
$ npm install @philholden/frog-poc-button

> @philholden/frog-poc-button@0.0.1 postinstall /Users/ph185068/projects/di/leapfrog/use-component/node_modules/@philholden/frog-poc-button
> builder run npm:postinstall

module.js:340
    throw err;
    ^

Error: Cannot find module '/Users/ph185068/projects/di/leapfrog/use-component/node_modules/@philholden/frog-poc-button/node_modules/builder-react-component/package.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Config._loadArchetypeScripts (/Users/ph185068/projects/di/leapfrog/use-component/node_modules/builder/lib/config.js:70:18)
    at null.<anonymous> (/Users/ph185068/projects/di/leapfrog/use-component/node_modules/builder/lib/config.js:98:26)
    at /Users/ph185068/projects/di/leapfrog/use-component/node_modules/lodash/index.js:2874:23
    at arrayMap (/Users/ph185068/projects/di/leapfrog/use-component/node_modules/lodash/index.js:1406:25)
    at map (/Users/ph185068/projects/di/leapfrog/use-component/node_modules/lodash/index.js:6710:14)
    at interceptor (/Users/ph185068/projects/di/leapfrog/use-component/node_modules/lodash/index.js:12240:26)
npm WARN EPACKAGEJSON leapfrog-poc@0.0.0 No description
npm WARN EPACKAGEJSON leapfrog-poc@0.0.0 No repository field.
npm ERR! Darwin 14.1.0
npm ERR! argv "/Users/ph185068/.nvm/versions/node/v5.1.0/bin/node" "/Users/ph185068/.nvm/versions/node/v5.1.0/bin/npm" "install" "@philholden/frog-poc-button"
npm ERR! node v5.1.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE

npm ERR! @philholden/frog-poc-button@0.0.1 postinstall: `builder run npm:postinstall`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @philholden/frog-poc-button@0.0.1 postinstall script 'builder run npm:postinstall'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @philholden/frog-poc-button package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     builder run npm:postinstall
npm ERR! You can get their info via:
npm ERR!     npm owner ls @philholden/frog-poc-button
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ph185068/projects/di/leapfrog/use-component/npm-debug.log
ph185068@WGBPH185068-5C1 use-component 
$ 
```

--- a/.builderrc
+++ b/.builderrc
@@ -1,3 +1,3 @@
 ---
 archetypes:
-  - builder-react-component
+  - builder-victory-component

--- a/package.json
+++ b/package.json
@@ -19,12 +19,12 @@
     "test": "builder run npm:test"
   },
   "dependencies": {
-    "builder": "2.1.0",
-    "builder-react-component": "0.1.0",
+    "builder": "^2.2.0",
+    "builder-victory-component": "~0.0.5",
     "radium": "^0.14.3"
   },
   "devDependencies": {
-    "builder-react-component-dev": "0.1.0",
+    "builder-victory-component-dev": "~0.0.5",
     "chai": "^3.2.0",
     "mocha": "^2.3.3",
     "react": "^0.14.0",

